dotCMS/src/main/webapp/html/portlet/ext/folders/order_menu.jsp

```diff
@@ -65,6 +71,30 @@ function goBack()
 	<% } %>
 }
 
+function emitErrorEvent() {

```

Comment:
Too much duplicated code... why don't just create a function where you can pass the event string?